<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QDialog" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <widget class="QWidget" name="habTab">
      <attribute name="title">
       <string>HAB</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="habSerialPortBox">
         <property name="title">
          <string>Serial Port</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,1">
          <item row="0" column="0">
           <widget class="QLabel" name="habSerialPortLabel">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="habSerialPortComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="habSerialSpeedLabel">
            <property name="text">
             <string>Speed</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="habSerialSpeedComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="habVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="localGpsTab">
      <attribute name="title">
       <string>Local GPS</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="localGpsSerialPortBox">
         <property name="title">
          <string>Local GPS Serial Port</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,0,1,0,0">
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="localGpsSerialSpeedComboBox"/>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="localGpsSerialDataBitsLabel">
            <property name="text">
             <string>Data bits</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="localGpsSerialParityLabel">
            <property name="text">
             <string>Parity</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="localGpsSerialParityComboBox"/>
          </item>
          <item row="0" column="1" colspan="5">
           <widget class="QComboBox" name="localGpsSerialPortComboBox"/>
          </item>
          <item row="3" column="3">
           <widget class="QComboBox" name="localGpsSerialDataBitsComboBox"/>
          </item>
          <item row="3" column="5">
           <widget class="QComboBox" name="localGpsSerialStopBitsComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="localGpsSerialSpeedLabel">
            <property name="text">
             <string>Speed</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLabel" name="localGpsSerialStopBitsLabel">
            <property name="text">
             <string>Stop bits</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="localGpsSerialPortLabel">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="localGpsSerialFlowControlLabel">
            <property name="text">
             <string>Flow Control</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QComboBox" name="localGpsSerialFlowControlComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="localGpsVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="serverTab">
      <attribute name="title">
       <string>Server</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="serverSyncBox">
         <property name="title">
          <string>Server Sync</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0">
          <item row="0" column="1">
           <widget class="QLineEdit" name="serverSyncAddressText"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="serverSyncAddressLabel">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="serverSyncPortLabel">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="serverSyncPortText"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="serverSyncNameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="serverSyncNameText"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="serverVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="logTab">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="logGroupBox">
         <property name="title">
          <string>Log storage</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QCheckBox" name="logRawCheckBox">
            <property name="text">
             <string>Store raw log</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="logRawPathText">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="logRawPathButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="logImagesCheckBox">
            <property name="text">
             <string>Keep images</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="logImagesPathText">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="logImagesPathButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="logVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>habSerialPortComboBox</tabstop>
  <tabstop>habSerialSpeedComboBox</tabstop>
  <tabstop>localGpsSerialPortBox</tabstop>
  <tabstop>localGpsSerialPortComboBox</tabstop>
  <tabstop>localGpsSerialSpeedComboBox</tabstop>
  <tabstop>localGpsSerialFlowControlComboBox</tabstop>
  <tabstop>localGpsSerialParityComboBox</tabstop>
  <tabstop>localGpsSerialDataBitsComboBox</tabstop>
  <tabstop>localGpsSerialStopBitsComboBox</tabstop>
  <tabstop>serverSyncBox</tabstop>
  <tabstop>serverSyncAddressText</tabstop>
  <tabstop>serverSyncPortText</tabstop>
  <tabstop>serverSyncNameText</tabstop>
  <tabstop>logRawCheckBox</tabstop>
  <tabstop>logRawPathText</tabstop>
  <tabstop>logRawPathButton</tabstop>
  <tabstop>logImagesCheckBox</tabstop>
  <tabstop>logImagesPathText</tabstop>
  <tabstop>logImagesPathButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
